pyodata/v2/service.py

```diff
@@ -858,6 +889,19 @@ def __getattr__(self, attr):
                 raise AttributeError('EntityType {0} does not have Property {1}: {2}'
                                      .format(self._entity_type.name, attr, str(ex)))
 
+    async def getattr(self, attr):
```

Comment:
I guess this method should have async_ prefix as well, similar to other new ones.

Will throw RuntimeError when called in synchronous script, where it looks like correct usage until seen the pyodata source code I guess.

```
import requests
import pyodata

import logging
logging.basicConfig(level=logging.DEBUG)

SERVICE_URL = 'http://services.odata.org/V2/Northwind/Northwind.svc/'

northwind = pyodata.Client(SERVICE_URL, requests.Session())
order = northwind.entity_sets.Order_Details.get_entity(OrderID=10248, ProductID=42).execute()
print(order.getattr('Quantity'))
```
```

<coroutine object EntityProxy.getattr at 0x000002032DFD7CA0>
...: RuntimeWarning: coroutine 'EntityProxy.getattr' was never awaited
  print(order.getattr('Quantity'))
```